content/content-strategy/manage-content-requests/create-content/content-decision-tree/index.yml

```diff
@@ -0,0 +1,15 @@
+pagetitle: Content decision tree
+description: A guide to help you decide if your content is right or ready to publish
+weight: 50
```

Comment:
Please check this weight value against all other index.yml for the module and parent Content strategy. It may be fine as is but I do recall more than one file with the same value and it may cause list order issues for the side nav.

Reply:
Trisha: resolved.